tests/vmi_gpu_test.go

```diff
@@ -79,7 +80,15 @@ var _ = Describe("[Serial][sig-compute]GPU", Serial, func() {
 			vmi, apiErr := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVMI)
 			Expect(apiErr).ToNot(HaveOccurred())
 
-			pod := libvmi.GetPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault)
+			By("Waiting for VMI to be scheduled")
+			Eventually(func() bool {
+				vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				return vmi.Status.Phase == v1.Scheduling
+			}, 480*time.Second, 1*time.Second).Should(BeTrue())
```

Comment:
This seems to fit `WaitForSuccessfulVMIStart*`. I would just use `WaitForSuccessfulVMIStartIgnoreWarnings` which has the default 180sec timeout embedded.
(480sec seems an extreme here)

Reply:
Aha, sorry for not noticing this one.